central/auth/datastore/datastore_impl_test.go

```diff
@@ -0,0 +1,240 @@
+//go:build sql_integration
+
+package datastore
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	pgStore "github.com/stackrox/rox/central/auth/datastore/internal/store/postgres"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/pkg/postgres/pgtest"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestAuthDatastorePostgres(t *testing.T) {
+	suite.Run(t, new(datastorePostgresTestSuite))
+}
+
+type datastorePostgresTestSuite struct {
+	suite.Suite
+
+	ctx       context.Context
+	pool      *pgtest.TestPostgres
+	store     pgStore.Store
+	datastore DataStore
```

Comment:
[no-op] This always confuses me, although I know that's the way we do it in our codebase.